.travis.yml

```diff
@@ -60,3 +57,8 @@ cache:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+  - git fetch --all && git config --global user.email "-" && git config --global user.name "-"
```

Comment:
did not add `gradlew clean build` here as it is unnecessary and will re-run the tests.